### DISCUSSION TITLE
How to claim ownership of a service?

### DISCUSSION THREAD

**Discussion by anderswassen** [EyevinnOSC/community#39] [Q&A] [answered]
Hi!

I recently launched an open source service in OSC. I am curious how i claim ownership of the service. In the Analytics tab I see the following instruction:

"Analytics only available when you have claimed ownership of services. Connect your GitHub account and the system will automatically connect the services you are the owner of and you can start...".

How do I claim ownership?

Keep up the great work!

**Comment by JoeldelPilar** [comment, marked as answer]
Hi @anderswassen 

Thank you for submitting your question to us and for using OSC!

To enable the claim functionality on the platform you need to connect your GitHub account to your tenant. Click on 'settings' on the left menu, then you go to the 'GitHub' tab and click on 'connect to GitHub'. When you are connected you will se a table with your GitHub accounts (probably only one). Default it's not activated but just click on the switch to start the auto claim flow.

For more info and step by step guide please visit: https://docs.osaas.io/osaas.wiki/Creator.html

Feel free to get back to us if theres any issues or questions!

Regards / Joel del Pilar

**Reply by anderswassen** [reply]
Awesome answer. It worked like a charm! Thanks!

May I suggest adding the link above to the Analytics tab? This could help avoid annoying questions from people like me. :-)

**Reply by JoeldelPilar** [reply]
That is a good idé, I'll create a issue for it and suggest it to the rest of the team. Thank you for good feedback! 🎉
